src/main/resources/config/application.yaml

```diff
@@ -1640,7 +1640,7 @@ notification:
                 representative:
                     emailId: 10e958ff-4ff4-4a38-b79c-298e89aa8079
                 joint_party:
-                    emailId: a3c3ace2-6b88-4557-b18a-7df1923a50f5
+                    emailId: 2542345a-3ac1-40a2-a22c-57a535b2b548
```

Comment:
need to update appellant and appointee for welsh appealDormant event as per ticket reqs

Reply:
done